spec/requests/posts_request_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'Get page post #index' do
+    before(:each) do
+      get '/users/64/posts'
+    end
+
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template('index')
+    end
+
+    it 'renders and get the placeholder text' do
+      expect(response.body).to include 'Here'
+    end
+  end
+
+  describe 'GET user page #show' do
+    before(:each) do
+      get '/users/64/posts/90'
+    end
+
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template('show')
+    end
+
+    it 'renders and get the placeholder text' do
+      expect(response.body).to include 'Here'
+    end
+  end
+end
```

Comment:
- Great job writing the test 👏🏽, Please note that some tests are failing, kindly fix them and you are through to the next project 💯.
![Screenshot from 2022-08-16 23-04-25](https://user-images.githubusercontent.com/44624138/184994553-3a1f2289-ddbc-474c-b8ba-04852e3cda62.png)